src/app/parcels/form/clientDatabaseFunctions.ts

```diff
@@ -31,15 +34,18 @@ export const insertParcel = async (
     if (error) {
         const logId = await logErrorReturnLogId("Error with insert: parcel data", error);
         await sendAuditLog({ ...auditLog, wasSuccess: false, logId });
-        return { error: { type: "failedToInsertParcel", logId } };
+        return { error: { type: "failedToInsertParcel", logId }, parcelId: undefined };
```

Comment:
can you use null instead of undefined to be consistent with how we return errors/data elsewhere?
(same for updateParcel)

Reply:
Yep happy to do this :)

Reply:
cool!